test/unit_tests/braket/circuits/test_ascii_circuit_diagram.py

```diff
@@ -442,8 +417,136 @@ def test_verbatim_qubset_qubits():
         "",
         "T  : |0|1|2|      3      |4|     5     |6|",
     )
-    expected = "\n".join(expected)
-    assert AsciiCircuitDiagram.build_diagram(circ) == expected
+    _assert_correct_diagram(circ, expected)
+
+
+def test_adjoint_1q():
+    circ = Circuit().s(0).rz(0, 3.14)
+    # Column formats to length of the gate plus the ascii representation for the angle.
+    expected = (
+        "T  : |     0      |  1  |",
+        "                         ",
+        "q0 : -(Rz(3.14))^†-(S)^†-",
```

Comment:
Rather than two characters it might look nicer if we just use an asterisk instead?

Reply:
While an asterisk is standard for adjoints in math, in physics it just means the complex conjugate, which is not what we want. I'm fine with just having the dagger without the ^ though